apps/web/src/hooks/useMonitoring.ts

```diff
@@ -9,8 +9,18 @@ export function useMonitoring() {
   const ldClient = useLDClient();
   const segment = useSegment();
 
+  const isNovuUser = currentUser && currentUser._id && !currentUser._id.startsWith('user_');
```

Comment:
checking `currentUser._id` and`currentOrganization._id` for undefined seems to be needed for e2e tests :/